autogpt_platform/backend/backend/executor/scheduler.py

```diff
@@ -166,7 +166,7 @@ def notification_client(self) -> NotificationManager:
 
     def run_service(self):
         load_dotenv()
-        db_schema, db_url = _extract_schema_from_url(os.getenv("DATABASE_URL"))
+        db_schema, db_url = _extract_schema_from_url(os.getenv("DIRECT_URL"))
```

Comment:
I'd like to pull this from settings. Is there a reason we can't? @majdyz

Reply:
There isn't, go fot it

Reply:
okay actually skipping to keep scope small